packages/model-viewer/src/three-components/gltf-instance/VariantMaterialLoaderPlugin.ts

```diff
@@ -0,0 +1,275 @@
+/**

```

Comment:
Please copy in the copyright header from any of our other files.

Reply:
It has to say Google, but you're welcome to add a comment about how it was derived from your own repro's code.

Reply:
Done